egs2/kss/tts1/README.md

```diff
@@ -29,6 +29,23 @@ See the following pages before asking the question:
 - [ESPnet2 Tutorial](https://espnet.github.io/espnet/espnet2_tutorial.html)
 - [ESPnet2 TTS FAQ](../../TEMPLATE/tts1/README.md#faq)
 
+# SECOND RESULTS
+- run by run2.sh with sampling frequency of 44,100 Hz
+
```

Comment:
Instead of making `run2.sh`, how about this one?
e.g., 
```
# SECOND RESULTS
- New G2P of `korean_jaso`
- New cleaner of `korean_cleaner`
- Sampling frequency of 44,100 Hz

# ```sh
# Run with the following command:
./run.sh \
    --fs 44100 \
    --fmin 80 \
    --fmax 22050 \
    --n_mels 120 \
    --n_fft 2048 \
    --n_shift 512 \
    --win_length 2048 \
    --train_config conf/tuning/train_tacotron2.yaml \
    --inference_config conf/tuning/decode_tacotron2.yaml \
    --token_type phn \
    --g2p korean_jaso \
    --cleaner korean_cleaner
# ```
```

(Please remove `# ` from `# ``` `.)

Reply:
Yeah sure. i will modify it this way.